apps/api/v2/src/ee/calendars/services/outlook.service.ts

```diff
@@ -156,7 +156,7 @@ export class OutlookService implements OAuthCalendarApp {
 
     const office365OAuthCredentials = await this.getOAuthCredentials(parsedCode);
 
-    const defaultCalendar = await this.getDefaultCalendar(accessToken);
+    const defaultCalendar = await this.getDefaultCalendar(office365OAuthCredentials.access_token);
```

Comment:
Here we need to pass in the access token for the users office365 oauth credentials.